src/function/scalar/string/mismatches.cpp

```diff
@@ -0,0 +1,47 @@
+#include "duckdb/function/scalar/string_functions.hpp"
+#include "duckdb/common/vector_operations/vector_operations.hpp"
+
+#include <ctype.h>
+#include <algorithm>
+
+namespace duckdb {
+
+static int64_t MismatchesScalarFunction(Vector &result, const string_t str, string_t tgt) {
+	int64_t num_characters_str = LengthFun::Length<string_t, int64_t>(str);
+	int64_t num_characters_tgt = LengthFun::Length<string_t, int64_t>(tgt);
+	if (num_characters_str != num_characters_tgt) {
+		return -1;  // Expected error code: both strings must be of equal length
```

Comment:
If you wish to throw an error from a function you should throw an exception, e.g. `throw InvalidInputException("...")`. This will report the error to the user.

Reply:
ok